flocker/node/agents/functional/test_ebs.py

```diff
@@ -315,11 +312,19 @@ class EBSProfiledBlockDeviceAPIInterfaceTests(
     pass
 
 
+class VolumeStub(object):
```

Comment:
Even if they're immutable, can you just *create* the real thing with the appropriate attributes? Creating and then setting attributes is kinda meh anyway, but with completely custom class it's even less useful  as a realistic test.